independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/jbang/JBangBuilderImpl.java

```diff
@@ -67,7 +67,7 @@ public static Map<String, Object> postBuild(Path appClasses, Path pomFile, List<
                         } else if (parts.length == 4) {
                             artifact = new ArtifactDependency(parts[0], parts[1], null, parts[2], parts[3]);
                         } else if (parts.length == 5) {
-                            artifact = new ArtifactDependency(parts[0], parts[1], parts[3], parts[2], parts[4]);
+                            artifact = new ArtifactDependency(parts[0], parts[1], parts[2], parts[3], parts[4]);
```

Comment:
I am not which artifact coords format JBang is using. Perhaps @maxandersen remembers.

Reply:
@aloubyansky: The format 
used is https://github.com/jbangdev/jbang/blob/main/src/main/java/dev/jbang/dependencies/DependencyUtil.java#L45

Reply:
Ignore this for now.

Reply:
There are more to it. The original code seems to respect the format, so the root cause lies elsewhere.

Reply:
Alright, it seems that jbang uses multiple formats for maven cooardinates but when it comes to integrations it uses what it calls canocial form, which is `groupId:artifactId:classifier:type:version` and is implemented here: https://github.com/jbangdev/jbang/blob/main/src/main/java/dev/jbang/dependencies/MavenCoordinate.java#L118

This form is definitely used by the IntegrationManager: https://github.com/jbangdev/jbang/blob/main/src/main/java/dev/jbang/spi/IntegrationManager.java#L73

I updated the PR with some comment that documents the above as it can get pretty confusing.

Reply:
GACTV is what we also use in Quarkus.